### PR TITLE
Fix more master image references in 1.13 files

### PR DESCRIPTION
**Pull request by ericvn**
Update `build-tools-centos` from master to release-1.13.

### CODE REVIEW

File: prow/config/jobs/proxy-1.13.yaml [hunk 1]
```diff
@@ -504,7 +504,6 @@ jobs:
   - presubmit
 - command:
   - ./prow/proxy-presubmit-centos-release.sh
-  image: gcr.io/istio-testing/build-tools-centos:master-2022-01-19T18-35-09
```
Comment (howardjohn) [inline prow/config/jobs/proxy-1.13.yaml:507]: this can't just be removed- needs to reference release-1.13, but not removed entirely?

Comment (ericvn) [reply prow/config/jobs/proxy-1.13.yaml:507]: The overall reference to 1.13 is on line 3.

Comment (ericvn) [reply prow/config/jobs/proxy-1.13.yaml:507]: I see centos now, which is why it wasn't found in searches in my last PR. Update coming.

Comment (ericvn) [reply prow/config/jobs/proxy-1.13.yaml:507]: Done.

**Review by howardjohn — APPROVED** [review]

**Pull request merged**